composer.json

```diff
@@ -9,6 +9,7 @@
     "ongr/elasticsearch-dsl": "^5.0.5",
     "pimcore/pimcore": ">=5.5.0 <6.1"
   },
+  "version": "1.10.0",
```

Comment:
Do we really need the version here? My feeling is, that we will always forget to update this. 

But I think, we don't need it ... can you duble-check that?

Reply:
@fashxp , thank for your comment and you right - this version isn`t used in trait (because ocramius/package-versions builds own Version.php based on tag@commit hash). 
PR has been updated.

Reply:
👍